lib/hanami.rb

```diff
@@ -118,6 +118,30 @@ def self.app
     App.new(configuration, environment)
   end
 
+  # Check to see if application allowed to load.
+  #
+  # You can set allowed application in `HANAMI_APPS` env variable.
```

Comment:
Suggestion: "The list of applications to be loaded can be set via the `HANAMI_APPS` env variable."